src/lib/shadow-shim-helper-rs/src/shim_event.rs

```diff
@@ -18,3 +22,48 @@ pub enum ShimEventID {
     AddThreadReq = 11,
     AddThreadParentRes = 12,
 }
+
+#[derive(Copy, Clone, Debug)]
+#[repr(C)]
+pub struct ShimEventSyscall {
+    pub syscall_args: SysCallArgs,
+}
+
+#[derive(Copy, Clone, Debug)]
+#[repr(C)]
+pub struct ShimEventSyscallComplete {
```

Comment:
I think these structs could use comments to explain what they are, or what they're used for.

Reply:
done